clusterman/mesos/mesos_cluster_connector.py

```diff
@@ -126,3 +126,6 @@ def _is_batch_framework(self, framework_name: str) -> bool:
         """If the framework matches any of the prefixes in self.non_batch_framework_prefixes
         this will return False, otherwise we assume the task to be a batch task"""
         return not any([framework_name.startswith(prefix) for prefix in self.non_batch_framework_prefixes])
+
+    def _freeze_agent(self, agent_id: str) -> None:
+        return
```

Comment:
i'd maybe add a comment here about how we could put the box in question into maintenance mode, but since we're almost done getting rid of mesos we're not putting in the effort to do so

Reply:
AFAIK we use is_cordoned:true to aws instance to maintain purpose also. So clusterman marks it not_killable: pool_manager -> _is_node_killable